Source/Core/buildModuleUrl.js

```diff
@@ -3,7 +3,7 @@ import DeveloperError from "./DeveloperError.js";
 import getAbsoluteUri from "./getAbsoluteUri.js";
 import Resource from "./Resource.js";
 
-/*global CESIUM_BASE_URL*/
+/*global CESIUM_BASE_URL,define,require*/
```

Comment:
just curious -- what does this do?

Reply:
nevermind, found it https://stackoverflow.com/questions/34183918/what-is-the-purpose-of-global-keyword-in-javascript-comments